Source/multi.cpp

```diff
@@ -197,17 +197,15 @@ void MonsterSeeds()
 
 void HandleTurnUpperBit(int pnum)
 {
-	int i;
-
-	for (i = 0; i < MAX_PLRS; i++) {
-		if ((player_state[i] & PS_CONNECTED) != 0 && i != pnum)
+	for (int i = 0; i < MAX_PLRS; i++) {

```

Comment:
```suggestion
	if (pnum == MyPlayerId) {
		gbDeltaSender = MAX_PLRS;
	}
	for (int i = 0; i < MAX_PLRS; i++) {
```

Reply:
Thanks, I specifically requested you for this PR cause I thought you might know if this was relevant :)